drivers/st7789/st7789.hpp

```diff
@@ -29,7 +29,9 @@ namespace pimoroni {
     uint bl     = SPI_BG_FRONT_PWM;
     uint vsync  = PIN_UNUSED; // only available on some products
 
-    static const uint32_t SPI_BAUD = 16 * 1000 * 1000;
+    // The ST7789 requires 16 ns between SPI rising edges.
```

Comment:
Should I explicitly call out that 16 ns is the writing speed, not the reading speed?

Reply:
I believe the "16" in "16 * 1000 * 1000" is from the 16 ns mentioned in the datasheet. So I do not think the existing code cares about the slower, unused 150 ns read speed.

If the existing code doesn't care, no need to add an extra concern here.